boskos/janitor/deployment.yaml

```diff
@@ -50,9 +51,10 @@ spec:
         image: gcr.io/k8s-testimages/janitor:v20180725-c421ed4f9
         args:
         - --boskos-url=http://boskos.test-pods.svc.cluster.local.
-        - --service-account=/etc/service-account/service-account.json
         - --resource-type=gce-project,gpu-project,ingress-project,istio-project,scalability-project
         - --pool-size=20
+        - --
+        - --service_account=/etc/service-account/service-account.json
```

Comment:
(separate PR, this is breaking before we bump the image)

Reply:
done